planner/core/cache.go

```diff
@@ -14,6 +14,7 @@
 package core
 
 import (
+	"github.com/pingcap/tidb/util/chunk"
```

Comment:
It seems a mistake :D

Reply:
Thank you for the comment. ^^; I've updated the PR.

Reply:
/run-all-tests